helix-term/src/config.rs

```diff
@@ -11,22 +11,14 @@ use toml::de::Error as TomlError;
 #[serde(deny_unknown_fields)]
```

Comment:
We use kebab case in the config:

```suggestion
#[serde(deny_unknown_fields, rename_all = "kebab-case")]
```